brickstrap.sh

```diff
@@ -461,14 +517,18 @@ function brp_configure_packages () {
         fi
     done
 
-    # run dpkg `--configure -a` twice because of errors during the first run
+    brp_fixup_etc_shells
+
     info "configuring packages..."
-    br_chroot_bind /usr/bin/dpkg --configure -a || \
-    br_chroot_bind /usr/bin/dpkg --configure -a || true
+    br_chroot_bind /usr/bin/dpkg --configure -a
```

Comment:
This is something from the original polystrap script. It is there because there were a number of packages that were found to fail on the first go around. It is quite possible that this may be needed in the future.